st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -212,8 +256,24 @@ def get_system_information():
     return system_information
 
 
-def create_archive(include_logs, include_configs, include_content, include_system_info,
-                   user_info=None, debug=False):
+def get_commands_output(config_yaml):
+    """"
+    Get output of the required shell command and redirect the output to a file.
+    :param config_yaml: config yaml file name
+    :return: output file paths
+    :rtype: ``list``
+    """
+    commands_list = get_config_details(config_yaml, 'shell_commands')
+    output_files_list = []
+    for cmd in commands_list:
+       output_file = "/tmp/%s.txt" % cmd
```

Comment:
Should probably do some additional formatting on the file name such as removing white spaces and special characters? Current implementation would result in a file called "rpm -qa.txt" right?